src/hist/plot.py

```diff
@@ -310,12 +314,14 @@ def func(
     # error bar keyword arguments
     eb_kwargs = _filter_dict(kwargs, "eb_")
     eb_kwargs.setdefault("label", "Histogram Data")
+    eb_kwargs.setdefault("fmt", "o")
```

Comment:
Yeah, I agree that the `"o"` maker looks much better visually then the previous defaults.